include/openssl/x509.h

```diff
@@ -2281,13 +2281,26 @@ OPENSSL_EXPORT int X509_REQ_print_fp(FILE *fp, X509_REQ *req);
 // XN_FLAG_FN_SN uses the attribute type's short name, when available.
 #define XN_FLAG_FN_SN 0
 
+// XN_FLAG_FN_LN uses the attribute type's long name, when available.
+#define XN_FLAG_FN_LN (1 << 21)
+
+// XN_FLAG_FN_OID always prints attribute types as OIDs.
+#define XN_FLAG_FN_OID (2 << 21)
+
+// XN_FLAG_FN_NONE skips printing field names.
+#define XN_FLAG_FN_NONE (3 << 21)
```

Comment:
looks like these [aren't used in ruby](https://github.com/search?q=repo%3Aruby%2Fruby+XN_FLAG_&type=code) or for defining `XN_FLAG_MULTILINE`. can we add another commit to this PR removing them? perhaps with a note explaining the gap in bit shift values.

Reply:
Sure I have no objection to that.

Reply:
Agreed that it's not necessary. Probably doesn't provide material benefit either.